### DISCUSSION TITLE
yarn generate doesn't generate the schema

### DISCUSSION THREAD

**Discussion by jjjtoikka** [wpengine/faustjs#437] [Q&A] [answered]
Hi guys,

Trying out faustjs with nextjs github starter repo. I started to add a new custom model to WP (with Atlas content modeler) and then run `yarn generate`, it took about 1 sec and nothing happened, the schema file didn't update. I tried deleting the file, and run it again, but the file is not generated. Also tried with `sudo`. Also tapping WP Admin -> GraphQL -> Enable Public Introspection doesn't help. Running it on MacOs BigSur.

Any ideas?

**Comment by jacobwise** [comment, marked as answer]
Are you getting any error output?

**Reply by jjjtoikka** [reply]
Hi,

Nope, it just shows that it run the script and everything is ok, but nothing happens. It also took a very short time (1 s) for script to run. Default models works, but if I make a new one it does not update it to the schema.

**Reply by blakewilson** [reply]
Hey @jjjtoikka, could you take a look at the release notes for Faust.js 0.10.0, specifically the breaking change around `gqty.config.js` and see if they help the issue you are seeing?

https://faustjs.org/docs/release-notes#faustjs-0100

**Reply by jjjtoikka** [reply]
Thanks @blakewilson 

I first updated the files as in 0.10.0 release notes said (in my 0.9.0) and then run `yarn generate` and it worked all ready with those changes.
Updated packages to the 0.10.2 and still worked nicely. I got all of the custom models to the schema and everything works (fingers crossed).

So problem solved!

cheers, J
